src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -31,9 +31,22 @@ public class WalletCreationRequest : RequestModel
     {
         public string Mnemonic { get; set; }
 
+        /// <summary>
+        /// This password is used to encrypt the wallet for secure storage. The password is required. 
+        /// If <see cref="Passphrase"/> is not supplied, then password is used as seed to the mnemonic.
+        /// </summary>
         [Required(ErrorMessage = "A password is required.")]
         public string Password { get; set; }
 
+        /// <summary>
+        /// When supplied, this passphrase is added as additional seed to the mnemonic.
+        /// </summary>
+        /// <remarks>
+        /// Empty string is a valid passphrase.
```

Comment:
"This passphrase is used as an additional seed (word) joined together with the <see cref="Mnemonic"/>."

Together with the remark:

"Empty string is a valid passphrase."

The passphrase is really just 1 additional word appended to the 12-word mnemonic, and that additional word is either "mnemonic" or "mnemonicMySpecialWordOrPhrase". So it is not entirely correct when you state that setting passphrase to "", it will become "mnemonic", as "mnemonic" is always prefixed to this seed no matter what the user supplies.

Sounds good with the example above, or needs additional work?

Reply:
Sounds great, you're right, leave it like that.